api/test/test_data/test_coverages_proto.json

```
@@ -2,43 +2,43 @@
     "observations": [
         {
             "ts_mdata": {
-                "title": "FMI test data",
-                "platform": "100945",
-                "standard_name": "TA_P1D_AVG",
-                "unit": "degC",
-                "instrument": "TA_P1D_AVG"
+                "platform": "0-20000-0-06257",
+                "standard_name": "air_temperature",
+                "unit": "degrees Celsius",
+                "level": "2.0",
+                "function": "mean",
+                "period": "PT1M",
+                "parameter_name": "air_temperature:2.0:mean:PT1M"
```

Should we add the timeseries_id to all of the test data for consistency and test expendability?

```suggestion
                "parameter_name": "air_temperature:2.0:mean:PT1M"
                "timeseries_id": "d036b10ac4f6e62049b9a496f04e0a82"
```